client/room.html

```diff
@@ -206,27 +206,36 @@ <h1>Share link</h1>
       <button id="shareOK">OK</button>
     </div>
 
-    <div id="aboutOverlay" class="overlay">
-      <h1>About</h1>
+    <div id="aboutOverlay" class="overlay" style = overflow:scroll>
+      <h1>About VirtualTabletop.io</h1>
       <p>This project is licensed under GPLv3 and is <a href="https://github.com/ArnoldSmith86/virtualtabletop">available on GitHub</a>.</p>
       <p>Come join our <a href="https://discord.gg/CEZz7wny9T">discord server</a>.</p>
-      <p>The default card deck was created by Adrian Kennard and released under CC0 Public Domain license: <a href="https://www.me.uk/cards/">https://www.me.uk/cards/</a></p>
+      <p>Check out our <a href="https://github.com/ArnoldSmith86/virtualtabletop/wiki">wiki page</a> for helpful information about how to make your own games.</p>
+      <p>So far 58.68 € of personal money has been spent to run this site. Donations have totaled 141.70 €. Thank you very much for you generosity. There are sufficient funds available to pay for the server for a few months, but please consider donating using PayPal to <a href="mailto:donate@virtualtabletop.io">donate@virtualtabletop.io</a>. 😀</p>
```

Comment:
This shouldn't be in the code. It's only added on the primary server whenever it updates. It doesn't make any sense if anyone starts their own server.

Reply:
As I see this is also the name of the project or do you mean the whole about page?

Reply:
The empty room overlay also has virtualtabletop.io in the title. not really sure whats the problem here.

Reply:
I didn't know the donations amount was not in the code.  I just copied what I saw from the main server.  It makes sense to me to not include that if you have another way you get that out there.  However, most of the information I've added "doesn't make any sense if anyone starts their own server."  But that isn't our audience for the most part.  I mean, why describe cautions on online gameplay and add a privacy policy either because if someone is on their own server, none of that matters.

Reply:
Oh ok so he was talking about the donations? I'll take a look into it. For the other stuff I think we can keep it and anyone is free to cut it out if they want.

Reply:
I think he was talking about the donations sentence, but he'll have to chime in if it was more than that.  And I agree with Raphael that we should keep this stuff.  If someone is sophisticated enough to start their own server with this, then they can cut out anything in the overlay they don't like.

Reply:
Yes, sorry, I was only talking about the donations. For the rest I agree with you two..